quickstarts/rhdh-installation-via-operator.yaml

```diff
@@ -139,26 +142,164 @@ spec:
       description: |-
         To create a Red Hat Developer Hub instance, you should create a `Backstage` custom resource.
 
-        1. If needed, switch or create a new project from the **Project dropdown**.
-        2. Click on the [perspective switcher]{{highlight qs-perspective-switcher}} at the top of the navigation and select **Administrator**.
-        3. In the Administrator perspective, go to the **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation.
+        1. Click on the [perspective switcher]{{highlight qs-perspective-switcher}} at the top of the navigation and select **Administrator**.
+        2. In the Administrator perspective, go to the **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation.
+        3. If needed, switch or create a new project from the **Project dropdown**.
         4. Select the "Red Hat Developer Hub" operator and navigate to the **Backstage** tab.
         5. Click on **Create Backstage** button in the top right corner of that tab view.
         6. **Optional:** Change the default configuration via the **Form view** or **YAML view** based on your needs.
+
+        **Alternative:** You can apply this YAML configuration also via the [Import YAML (plus icon)]{{highlight qs-masthead-import}} button at the top of the navigation.
+
+        A minimal YAML configuration could look like this:
+
+        ```
+        apiVersion: rhdh.redhat.com/v1alpha1
+        kind: Backstage
+        metadata:
+          name: my-rhdh
+          labels:
+            app.kubernetes.io/instance: my-rhdh
+            app.kubernetes.io/name: backstage
+        spec:
+          application:
+            image: quay.io/rhdh/rhdh-hub-rhel9
+        ```{{copy}}
+        <br/>
+
+        The following steps will give you some examples of how to customize your RHDH instance.
+
+        Press **Create** to create a new Red Hat Developer Hub instance.
```

Comment:
Also, this seems like a step and gets buried under the example YAML configuration. I think it would help if it had a step number by it. Would this be considered step 7 maybe?